### ISSUE TITLE
Graham Project 4

### ISSUE THREAD

**Issue by Detest9067** [Ironhack-data-bcn-january-2023/project-IV-sql-tableau#6]
https://github.com/Detest9067/Project-IV-SQl-Tableau

**Comment by ta-data-bcn** [comment]
Heeelloo Graham 🚀

Great job on the project, see below for comments :) 

**Organization of your files** 📚

- Your .gitignore works, but you could also add the __pycache__ and the .ipynb_checkpoints
- I would add a src folder for the .py files and an images folder.

**README.md** ✍🏽

- Your README is okaaay, short but gets to the point of what you did and the libraries/tools you used.
- You could've added some screenshots of the Tableau graphs, aparte from the link. Not all, but a couple, just to show a little bit of your results. 

**The code** 👨🏽‍💻

- I loooove the imgscraper, really nice job. I would be cool if you added some of those pictures to your visualizations. Something like [this](https://public.tableau.com/app/profile/datasaurus.rex/viz/GameofThronesDeathsVizMkIII/Main) 👇🏾

![Captura de Pantalla 2023-03-03 a la(s) 13 15 58](https://user-images.githubusercontent.com/47005065/222717697-db32eede-5287-4416-be49-e81c13ebd364.png)

- Your SQL queries look good, they're simple but enough to get what you wanted. 
- In general your code is okay. Great job using an API, and I guess your csv file was pretty clean so you didn't have to do a lot of cleaning and rearranging, which was not the goal of the project, so niceeee. 

**Viz** 🎨 

- In general your graphs look great. Really nice dashboards Graham 😊
- I love that in the Character Overview you can choose the characters you want to add to the table :) 
- It would be cool to see the change in polarity for the same character across the seasons. 
- I would change the "House AVG Line" for another type of plot. Maybe a barplot works better. 

In conclusion, your project looks good Graham. Alsoooo, I love Game of Thrones so I really liked the topic. 

Congrats and good luck on the last project, I am sure you'll do a great job.
